minimal-launchpad/minimal_ui_index.js

```diff
@@ -46,6 +46,7 @@ let device = null;
 let transport;
 let chip = "default";
 let chipDesc = "default";
+let consoleBaudrateFromToml;
```

Comment:
should this have a default value 115200? if we don't get the value from toml?

Reply:
I think there is no need of that as if we don't get any value from toml, by default it will become undefined by JS engine and that will get handled at **transport.connect(consoleBaudrateFromToml)**
When **consoleBaudrateFromToml = undefined** then the default value of baud = 115200 will be considered which is present at esptool-js **connect method of transport class**.